src/engine/mod.rs

```diff
@@ -299,7 +299,8 @@ impl Engine {
         self.ui_time = instant::Instant::now() - time;
     }
 
-    pub(crate) fn handle_model_events(&mut self) {
+    /// Handle hot-reloading of resources.
```

Comment:
It is worth noting that this method should not be called manually if one use `Engine::update`